client/packages/system/src/Patient/Encounter/CreateEncounterModal.tsx

```diff
@@ -81,7 +79,10 @@ export const CreateEncounterModal: FC = () => {
   };
 
   const setStartDatetime = (date: Date | null): void => {
-    const startDatetime = DateUtils.formatRFC3339(date);
+    date?.setTime(date.getTime() + 50000);
```

Comment:
Why add 50000 here?

Reply:
Oops, that's leftover from my experimenting.

Removed: 385729b87